src/libsyncengine/jobs/network/API_v2/upload_session/uploadsessionstartjob.cpp

```diff
@@ -58,7 +58,6 @@ void UploadSessionStartJob::setData(bool &canceled) {
             if (_fileId.empty()) {
                 json.set("file_name", _filename);
                 json.set("directory_id", _remoteParentDirId);
-                json.set("conflict", "version");
```

Comment:
Yes, because with this flag, if there are 2 files with the same name but different encodings, the back will process the upload as an edit, not a creation. Therefor, the request succeeds, but then the desktop app will get an error when trying to insert the new item in DB.
Without this flag, the back will return an error on item creation and the app will blacklist the item immediately to avoid further errors.